plugins/modules/aci_l4l7_service_graph_template_abs_connection.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_service_graph_template_abs_connection
+short_description: Manage L4-L7 Service Graph Template Abs Connections (vns:AbsConnection)
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Service Graph Template  Connections.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  service_graph:
+    description:
+    - The name of an existing Service Graph.
+    type: str
+  connection_name:
+    description:
+    - The name of the connection.
+    type: str
+  direct_connect:
+    description:
+    - Whether to enable direct connections.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  unicast_route:
+    description:
+    - Whether to enable unicast routing.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+  adjacency_type:
+    description:
+    - Whether the adjacecncy is Layer2 or Layer3.
+    - The APIC defaults to C(l2) when unset during creation.
+    type: str
+    choices: [ l2, l3 ]
+  connector_direction:
+    description:
+    - The connector direction.
+    type: str
+    choices: [ provider, consumer ]
```

Comment:
I see default value is unknown, is this a valid option?

Reply:
Changed it